src/main/java/bio/terra/service/filedata/flight/delete/FileDeleteFlight.java

```diff
@@ -44,10 +46,8 @@ public FileDeleteFlight(FlightMap inputParameters, Object applicationContext) {
         //  We should NOT put code like that in the flight constructor.
         Dataset dataset = datasetService.retrieve(UUID.fromString(datasetId));
 
-        RetryRuleRandomBackoff fileSystemRetry =
-            new RetryRuleRandomBackoff(500, appConfig.getMaxStairwayThreads(), 5);
-        RetryRuleRandomBackoff lockDatasetRetry =
-            new RetryRuleRandomBackoff(500, appConfig.getMaxStairwayThreads(), 5);
+        RetryRuleRandomBackoff fileSystemRetry = getDefaultRandomBackoffRetryRule(appConfig.getMaxStairwayThreads());
+        RetryRuleRandomBackoff lockDatasetRetry = getDefaultRandomBackoffRetryRule(appConfig.getMaxStairwayThreads());
```

Comment:
Is there a reason why one rule can't be used for both? It looks like rules are `initialize()`'d before they're used. (Although, it does seem odd to me that a rule is stateful; maybe another way to do this would've been to make rules stateless and have the Flight code create state for a rule when needed.)

Reply:
Yeah, these are in fact stateful objects so each retry rule needs it's own instance (which is why I made the factory methods as opposed to just having static methods